src/plugins_exts_compile.h

```diff
@@ -99,14 +99,15 @@ const char *lysc_ctx_get_path(const struct lysc_ctx *ctx);
  * (or if just the @ref scflags are enough to modify the compilation process).
  *
  * @param[in] ctx YANG schema compile context to track the compilation state.
- * @param[in] ext_p Parsed representation of the extension instance being processed.
+ * @param[in,out] ext_p Parsed representation of the extension instance being processed.
+ * Item ::lysc_ext_instance.parsed will be set.
  * @param[in,out] ext Compiled extension instance with the prepared ::lysc_ext_instance.substmts array, which will be updated
  * by storing the compiled data.
  * @return LY_SUCCESS on success.
  * @return LY_EVALID if compilation of the substatements fails.
  * @return LY_ENOT if the extension is disabled (by if-feature) and should be ignored.
  */
-LY_ERR lys_compile_extension_instance(struct lysc_ctx *ctx, const struct lysp_ext_instance *ext_p, struct lysc_ext_instance *ext);
+LY_ERR lys_compile_extension_instance(struct lysc_ctx *ctx, struct lysp_ext_instance *ext_p, struct lysc_ext_instance *ext);
```

Comment:
And leave `const` here as well, just use a cast inside the function to get rid of it.